examples/certificate-check-volume.yaml

```diff
@@ -0,0 +1,48 @@
+# An example Job to run a certificate check of OpenShift's internal
+# certificate status from within OpenShift.
+#
+# It uses the openshift/openshift-ansible container image.
+# (see README_CONTAINER_IMAGE.md in the top level dir for more details).
+#
+# It expects:
+#   - the Ansible inventory file from the 'hosts' key of a ConfigMap
+#     named 'inventory'
+#   - the ssh key in the 'ssh-privatekey' key of a Secret named 'sshkey'
+#   - a PersistentVolumeClaim named 'certcheck-reports' where the
+#     generated reports are stored
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: certificate-check
+spec:
+  containers:
+  - name: openshift-ansible
+    image: openshift/openshift-ansible
+    env:
+    - name: PLAYBOOK_FILE
+      value: playbooks/certificate_expiry/html_and_json_timestamp.yaml
```

Comment:
there's also an extra volume in the container spec.

I thought about providing a template to cover various use cases - it does get tricky though and it requires more though... for example, the template would be expected to pack all the required resources, and this would include the configmap and secrets, right?

In other words: I left the template for later...